frontend_app_folder_advanced/src/container/FolderAdvanced.jsx

```diff
@@ -106,11 +109,17 @@ export class FolderAdvanced extends React.Component {
     props.appContentCustomEventHandlerReloadAppFeatureData(this.loadContent, this.loadTimeline, this.buildBreadcrumbs)
   }
 
-  handleFolderChanged = message => {
+  handleFolderChanged = data => {
     const { state } = this
-    if (message.content.content_id !== state.content.content_id) return
-    this.setState({ content: message.content })
-    this.setHeadTitle(message.content.label)
+    if (data.content.content_id !== state.content.content_id) return
+
+    this.setState(prev => ({
+      content: prev.loggedUser.userId === data.author.user_id ? { ...prev.content, ...data.content } : prev.content,
+      newContent: { ...prev.content, ...data.content },
+      editionAuthor: data.author.public_name,
+      hasUpdated: prev.loggedUser.userId !== data.author.user_id
```

Comment:
it seems that the name `hasUpdated` means that the logged user did the update we just received.
I would like a more explicit property name then

Reply:
EDIT:
I'm now seeing that this property is only an addition in this app and that it already is named `hasUpdated` in others.
I'd still like you to rename them all but if it is too complicated, you can keep it that way.
I should have seen it in the first PR adding it. It's not a big issue anyway

Reply:
Pretty easy to change, a little less easy to think of a good name. I'll change it to `receivedUpdate`, but if you have an idea of a better name, you can say I don't mind changing it again.

Reply:
mmh since it is related to the logged user, I'd like to reflect that info.
`isUpdateFromLoggedUser`, `loggedUserHasUpdated`, isAuthorLoggedUser` 
Don't be afraid of long variable names, especially when they are "kinda" global (it's not global here by it is global to the app)

Reply:
I see you already did the refactor, sorry to answer that late

Reply:
In fact, it depends, for this component it is related to the loggedUser, but for the agenda for example it is related to which page the user is on. Is it better to have more specific but different names between the components or the same name, since the variable has the same function, but more generic?

Reply:
tough question.
I think since the apps are stand alone React apps we can allow different variable names of the same job.
Ideally, the name is different enough to clearly reflect its usage in each apps and similar enough to understand that these variable are equivalents in each apps.
Good luck with that : )

Reply:
and if I put `showRefreshWarning`, do you like it? It seems to me more descriptive about the variable's purpose and works for all components.

Reply:
I think it's great. As long as it is actually only used for that purpose

Reply:
so we have a winner! I will update the code